lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -335,7 +335,7 @@ class DetailsRenderer {
 
   /**
    * @param {LH.Audit.Details.List} details
-   * @returns {Element}
+   * @return {Element}
```

Comment:
fwiw http://usejsdoc.org/tags-returns.html

Reply:
> fwiw http://usejsdoc.org/tags-returns.html

I don't feel bad about us standardizing on one of these :)

Reply:
Oh I don't feel bad about standardizing on one either :) 

but I couldn't stand to left `@returns` be called incorrect when in fact it is the actual "spec" one